setup.cfg

```diff
@@ -1,6 +1,6 @@
 [pytest]
 DJANGO_SETTINGS_MODULE=test_settings
-addopts = --verbose --nomigrations --reuse-db -s --ignore ve --flake8 --cov --nomigrations
+addopts = --verbose --nomigrations --reuse-db -s --ignore ve --flake8
```

Comment:
IIRC, it's really just the Hypothesis tests that are slow (and that's more the Django ORM's fault than anything else). Does turning off coverage give a noticeable speedup?